taxi/admin.py

```diff
@@ -0,0 +1,24 @@
+from django.contrib import admin
+from django.contrib.auth.admin import UserAdmin
+
+from taxi import models
+
+
+@admin.register(models.Driver)
+class DriverAdmin(UserAdmin):
+    list_display = UserAdmin.list_display + ("license_number",)
+    fieldsets = UserAdmin.fieldsets + (
+        ("Additional info", {"fields": ("license_number",)}),
+    )
+    add_fieldsets = UserAdmin.add_fieldsets + (
+        ("Additional info", {"fields": ("license_number",)}),
+    )
+
+
+@admin.register(models.Car)
+class CarAdmin(admin.ModelAdmin):
+    list_filter = ["manufacturer"]
+    search_fields = ["model"]
+
+
+admin.site.register(models.Manufacturer)

```

Comment:
You should choose one approach for registering your models here, either functional or with the use of decorators.